OrbitLinuxTracing/include/OrbitLinuxTracing/Events.h

```diff
@@ -53,6 +53,7 @@ class CallstackFrame {
   const std::string& GetFunctionName() const { return function_name_; }
   uint64_t GetFunctionOffset() const { return function_offset_; }
   const std::string& GetMapName() const { return map_name_; }
+  static constexpr uint64_t kUnknownFunctionOffset  = static_cast<uint64_t>(-1);
```

Comment:
Why do you cast from -1 here (into an unsigned type)? Shouldn't you get the same by doing std::numeric_limits<uint64_t>::max()? The latter would be more obvious regarding the intention here.

Reply:
I switched to `std::numeric_limits<uint64_t>::max()`